app/tools/simulation/mace/jobs/runner.py

```
@@ -0,0 +1,347 @@
+            push_url = provmod.push_to_dataset(
+                cache_key,
+                files={
+                    "provenance.json": self.cache.root / cache_key / "provenance.json",
+                    "result.json": self.cache.root / cache_key / "result.json",
+                    "structure.cif": self.cache.root / cache_key / "structure.cif",
+                    "traj.json": self.cache.root / cache_key / "traj.json",
+                },
+            )
+            if push_url:
+                self.store.set_provenance_ref(job_id, push_url)
+        except Exception as ex:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)